components/supervisor/frontend/src/index.ts

```diff
@@ -284,6 +287,7 @@ LoadingFrame.load().then(async (loading) => {
             IDEWebSocket.connectWorkspace(),
             frontendDashboardServiceClient.onInfoUpdate((status) => {
                 if (status.statusPhase === "stopping" || status.statusPhase === "stopped") {
+                    maybeRedirectToCustomUrl(frontendDashboardServiceClient.latestInfo);
```

Comment:
if the redirect is enabled and works, we should not show the stopping screen at all.